### PR TITLE
Brat extension

### PR DESCRIPTION
**Pull request by dandersonw**

### CODE REVIEW

**Comment by dandersonw** [comment]
I am provisionally satisfied.

File: ai2_brat/filelock.py [hunk 1]
```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python
```
Comment (wammar) [inline ai2_brat/filelock.py:1]: Why do we need this?

Comment (wammar) [reply ai2_brat/filelock.py:1]: Ah, this must be the python3 versions.

**Review by wammar — APPROVED** [review]
LGTM. Would it be a good idea to move the python3 version of modules originally included in brat in a separate subdirectory? This way we can easily tell apart original vs. custom modules.

**Comment by dandersonw** [comment]
I don't think it's necessary to create a subdirectory. All of the files that were originally part of brat have comments at the top suggesting their origin.

**Pull request merged**